content/docs/authorization/cli.md

```diff
@@ -1095,3 +1096,42 @@ karavictl tenant delete [flags]
 $ karavictl tenant delete --name Alice
 ```
 On success, there will be no output. You may run `karavictl tenant get --name <tenant-name>` to confirm the deletion occurred.
+
+
+
+---
+
+
+
+### karavictl tenant update
+
+Updates a tenant's approvesdc boolean flag value within CSM only for Powerflex Pre-approved Guid feature
+
+##### Synopsis
+
+Updates a tenant resource within CSM (currently only for powerflex)
+
+```
+karavictl tenant update [flags]
+```
+
+#### Options
+
+```
+  -h, --help   help for create
+  -n, --name string   Tenant name
+      --approvesdc boolean (Usage: --approvesdc=true/false | true value will allow the tenant's request to approve SDC pass through proxy server to powerflex array. false value will block the request to approve SDC if SDC is not in approved state)
+```
+
+##### Options inherited from parent commands
+
+```
+      --addr string     Address of the server (default "localhost:443")
+      --config string   config file (default is $HOME/.karavictl.yaml)
+```
+
+##### Output
+```
+$ karavictl tenant update --name Alice --approvesdc=false
+```
+On success, there will be no output. You may run `karavictl tenant get --name <tenant-name>` to confirm the deletion occurred.
```

Comment:
'to confirm the deletion occurred' should be changed to 'confirm the update was persisted'

Reply:
made the changes